src/main/java/me/coley/recaf/config/Config.java

```diff
@@ -34,8 +35,11 @@ public String getName() {
 	}
 
 	@SuppressWarnings("unchecked")
-	void load(File file) throws IOException {
-		final JsonObject json = Json.parse(FileUtils.readFileToString(file, UTF_8)).asObject();
+	void load(Path file) throws IOException {
+		final JsonObject json;
+		try (BufferedReader bufferedReader = Files.newBufferedReader(file)) {
+			json = Json.parse(bufferedReader).asObject();
+		}
```

Comment:
Instead of this, the original can be used with only the small modification of `file` -> `file.toFile()`
Also, with the type change, the parameter name should change to something like `path`